app/models/course/assessment/answer.rb

```diff
@@ -2,7 +2,7 @@
 class Course::Assessment::Answer < ActiveRecord::Base
   include Workflow
   actable
-  acts_as :discussion_topic, class_name: Course::Discussion::Topic.name
+  acts_as_discussion_topic display_global: true
```

Comment:
Actually I prefer code to be self-explainable and less comments, but in this case, `global` looks good to me.